City-of-Helsinki/drupal-infofinland
# UHF-10468: removed purge & varnish packages

# [UHF-10468](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10468)
Remove varnish & purge modules

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-10468`
  * `make fresh`
* Run `make drush-cr`

## How to test
Check that make fresh works and site is up and running and status page works



[UHF-10468]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10468?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
